records_mover/records/targets/data_url.py

```diff
@@ -63,6 +63,13 @@ def move_from_records_directory(self,
                               records_format.generate_filename('data'): self.output_loc.url
                           })
 
+    def can_move_from_temp_loc_after_filling_it(self) -> bool:
+        # we can always create a temporary directory
+        return True
+
+    # TODO: Why even does this implement this protocol?  Can I go
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker